tests/tests_integration/test_api/test_templates.py

```diff
@@ -83,23 +83,26 @@ def new_template_instance(new_template_group_version):
 def new_view(new_template_group_version):
     events = []
     for i in range(0, 1001):
-        events.append(Event(external_id="test_evt_templates_" + uuid.uuid4().hex[0:20], type="test_templates", start_time=i * 1000))
-    API.events.create(events)
+        events.append(Event(external_id="test_evt_templates_1_" + str(i), type="test_templates_1", start_time=i * 1000))
+    try:
+        API.events.create(events)
+    except:
+        # We only generate this data once for a given project, to prevent issues with eventual consistency etc.
```

Comment:
think I missed this comment earlier 😅

Reply:
Hehe, I didn't have the comment before :)